config/initializers/delayed_job.rb

```diff
@@ -36,10 +46,7 @@
     'NameError' => ALWAYS_FAIL,
     'NoMethodError' => ALWAYS_FAIL,
     'NotYetImplemented' => ALWAYS_FAIL,
-    # http://stackoverflow.com/a/31928089
-    'ActiveJob::DeserializationError' => ->(exception) do
-      exception.original_exception.is_a? ActiveRecord::RecordNotFound
-    end,
+    'ActiveJob::DeserializationError' => ALWAYS_FAIL,
```

Comment:
No longer takes params so I (had to) set it to always fail with background jobs.